src/hooks/index.ts

```diff
@@ -1,94 +1,66 @@
-import { Web3Provider } from '@ethersproject/providers'
-import { ChainId } from '@swapr/sdk'
-
-import { UnsupportedChainIdError, useWeb3React as useWeb3ReactCore } from '@web3-react/core'
-import { Web3ReactContextInterface } from '@web3-react/core/dist/types'
 import { useEffect, useState } from 'react'
-import { isMobile } from 'react-device-detect'
-
-import { injected } from '../connectors'
-import { NetworkContextName } from '../constants'
 
-export function useActiveWeb3React(): Web3ReactContextInterface<Web3Provider> & { chainId?: ChainId } {
-  const context = useWeb3ReactCore<Web3Provider>()
-  const contextNetwork = useWeb3ReactCore<Web3Provider>(NetworkContextName)
-  return context.active ? context : contextNetwork
-}
+import { web3Network } from '../connectors'
+import { useWeb3ReactCore } from './useWeb3ReactCore'
 
 export function useEagerConnect() {
-  const { activate, active } = useWeb3ReactCore() // specifically using useWeb3ReactCore because of what this hook does
+  const { isActive } = useWeb3ReactCore() // specifically using useWeb3ReactCore because of what this hook does
   const [tried, setTried] = useState(false)
 
+  // attempt to connect eagerly on mount
   useEffect(() => {
-    injected.isAuthorized().then(isAuthorized => {
-      if (isAuthorized) {
-        activate(injected, undefined, true).catch(() => {
-          setTried(true)
-        })
-      } else {
-        if (isMobile && window.ethereum) {
-          activate(injected, undefined, true).catch(() => {
-            setTried(true)
-          })
-        } else {
-          setTried(true)
-        }
-      }
+    void web3Network.activate().catch(() => {
+      console.debug('Failed to connect to network')
     })
-  }, [activate]) // intentionally only running on mount (make sure it's only mounted once :))
+  }, [])
 
   // if the connection worked, wait until we get confirmation of that to flip the flag
   useEffect(() => {
-    if (active) {
+    if (isActive) {
       setTried(true)
     }
-  }, [active])
+  }, [isActive])
 
   return tried
 }
 
-/**
- * Use for network and injected - logs user in
- * and out after checking what network theyre on
- */
-export function useInactiveListener(suppress = false) {
-  const { active, error, activate } = useWeb3ReactCore() // specifically using useWeb3React because of what this hook does
-
-  useEffect(() => {
-    const { ethereum } = window
+// /**
+//  * Use for network and injected - logs user in
+//  * and out after checking what network theyre on
+//  */
+// export function useInactiveListener(suppress = false) {
```

Comment:
should we keep this code commented?

Reply:
Removed [8dcd3a8](https://github.com/levelkdev/swapr-dapp/pull/1330/commits/8dcd3a800475eee37a700487e34037ba9150033f)